data_prep/introspector.py

```diff
@@ -42,7 +42,11 @@
 def _query_introspector(api: str, params: dict) -> dict:
   """Queries FuzzIntrospector API and return data specified by |key|,
   returns None if unable to get the value."""
-  resp = requests.get(api, params, timeout=TIMEOUT)
+  try:
+    resp = requests.get(api, params, timeout=TIMEOUT)
+  except requests.exceptions.Timeout as err:
+    logging.error('Failed to get data from FI due to timeout\n%s', err)
+    return {}
```

Comment:
1. It would be better to use the 'delay and retry' trick again here, just in case this is caused by too frequent API access.
2. Could you please log more details about the request to help FI debug. For example, `api`, `params`.

Reply:
Done. Moving this change to new pr #128 as it's more than a few lines